geomstats/learning/embedding_data.py

```diff
@@ -0,0 +1,242 @@
+"""Learning embeddings of data on manifolds."""
+
+import logging
+
+import geomstats.backend as gs
+from geomstats.geometry.poincare_ball import PoincareBall
+
+
+class Embedding ():
+    """A class for learning embeddings of data on manifolds.
+
+    Currently supports embedding of graph-structured data on
```

Comment:
No, as it is today it would need rework to generalize it to other manifolds, changed data to graph and set manifold as PoincareBall by default, only dim is given to the constructor.